xml/security_ldap_migrate_test.xml

```diff
@@ -0,0 +1,181 @@
+     is <replaceable>dc=ldap1,dc=com</replaceable>. If you are using the 
+     <filename>/etc/openldap/slapd.conf</filename>
+     format, use the following command to create the ldif backup file:
+   </para>
+   <screen>&prompt.sudo;slapcat -f /etc/openldap/slapd.conf -b <replaceable>dc=ldap1,dc=com</replaceable> -l /root/<replaceable>ldap1-com</replaceable>.ldif
+   </screen>
+   <para>
+     For the <filename>/etc/openldap/slapd.d</filename> format use the following command:
+   </para>
+   <screen>&prompt.sudo;slapcat -f /etc/openldap/slapd.conf -b <replaceable>dc=ldap1,dc=com</replaceable> -l /root/<replaceable>ldap1-com</replaceable>.ldif</screen>   
+   <para>
+     Use <command>openldap_to_ds</command> to import the ldif files:
+   </para>
+   <screen>&prompt.sudo;openldap_to_ds <replaceable>ldap1</replaceable> /root/slapd.d <replaceable>ldap1-com</replaceable>.ldif</screen>
+   <para>
+     This performs a dry-run and does not change anything. The output looks
+     like this:
+   </para>
+   <screen>Examining OpenLDAP Configuration ...
+Completed OpenLDAP Configuration Parsing.
+Examining Ldifs ...
+Completed Ldif Metadata Parsing.
+The following migration steps will be performed:
+ * Schema Skip Unsupported Attribute -> otherMailbox (0.9.2342.19200300.100.1.22)
+ * Schema Skip Unsupported Attribute -> dSAQuality (0.9.2342.19200300.100.1.49)
+ * Schema Skip Unsupported Attribute -> singleLevelQuality (0.9.2342.19200300.100.1.50)
+ * Schema Skip Unsupported Attribute -> subtreeMinimumQuality (0.9.2342.19200300.100.1.51)
+ * Schema Skip Unsupported Attribute -> subtreeMaximumQuality (0.9.2342.19200300.100.1.52)
+ * Schema Create Attribute -> suseDefaultBase (SUSE.YaST.ModuleConfig.Attr:2)
+ * Schema Create Attribute -> suseNextUniqueId (SUSE.YaST.ModuleConfig.Attr:3)
+[...]
+ * Schema Create ObjectClass -> suseDhcpConfiguration (SUSE.YaST.ModuleConfig.OC:10)
+ * Schema Create ObjectClass -> suseMailConfiguration (SUSE.YaST.ModuleConfig.OC:11)
+ * Database Reindex -> dc=example,dc=com
+ * Database Import Ldif -> dc=example,dc=com from example.ldif - excluding entry attributes = [{'structuralobjectclass', 'entrycsn'}]
+No actions taken. To apply migration plan, use '--confirm'
+</screen>
+   <para>
+     The following example performs the migration, and the output looks 
+     different from the dry-run:
+   </para>
+   <screen>&prompt.sudo;openldap_to_ds <replaceable>ldap1</replaceable> /root/slapd.d <replaceable>ldap1-com</replaceable>.ldif --confirm
+Starting Migration ... This may take some time ...
+migration: 1 / 40 complete ...
+migration: 2 / 40 complete ...
+migration: 3 / 40 complete ...
+[...]
+Index task index_all_05252021_120216 completed successfully
+post: 39 / 40 complete ...
+post: 40 / 40 complete ...
+🎉 Migration complete!
+----------------------
+You should now review your instance configuration and data:
+ * [ ] - Create/Migrate Database Access Controls (ACI)
```

Comment:
You asked for some examples of these, but ... well, if it was that easy I'd just do it in the migration! For aci/tls, these would probably end up being links to those chapters of docs in the future, but that's something that the administrator has to devise and develop. Automatic backups is up to the admin and how they backup software works and integrates to trigger 389 to do the backup for example. So It's hard to give "examples" here because they are so subjective.

Reply:
Eh, I misremembered, I thought the output from your demos was longer after 'Migration Complete'. Never mind.